content/livesync/quickstart.textile

```diff
@@ -1,7 +1,228 @@
+      }
+      const [data, sequenceID] = await getPost(id);
+      return NextResponse.json({ sequenceID, data });
+    } catch (error) {
+      return NextResponse.json({ message: 'failed to get post', error }, { status: 500 });
+    }
+  }
+```
+
+h4(#outbox-entry). outbox entry.
+
+The outbox entry is an entry of an update to one of the models within the database. In "this example":https://github.com/ably-labs/models/blob/main/examples/livecomments/lib/prisma/api.ts#L129-L139 it's a record of a comment being added to, updated with, or deleted from a post. When a post is added, updated, or deleted a record of this is also added to the outbox table so that the ADBC can pick this up and publish the update to the specified Pub/Sub channel.
+
+```[javascript]
+  export async function withOutboxWrite(
+    op: (tx: TxClient, ...args: any[]) => Promise<Prisma.outboxCreateInput>,
+    ...args: any[]
+  ) {
+    return await prisma.$transaction(async (tx) => {
+      const { mutation_id, channel, name, data, headers } = await op(tx, ...args);
+      await tx.outbox.create({
+        data: { mutation_id, channel, name, data, headers },
+      });
+    });
+  }
+```
+
+h2(#quickstart). Quickstart
+
+In this quickstart you will cover all the steps required to get an existing demo running locally on your machine.
+
+h3(#step-0). The project
+
+Clone the models repository (https://github.com/ably-labs/models/) and navigate to the @livecomments@ directory within @examples@. The @livecomments@ directory houses the project used for this working demo.
+
+h3(#step-1). Create an Ably account
+
+"Sign up":https://ably.com/signup for a free account to get your own API key. Ensure that your API key includes the @subscribe@ and @publish@ "capabilities":/auth/capabilities.
+
+h3(#step-2). Update environment variables
+
+Set up environment variables by copying them from the template:
+
+```
+cp env.example env.local
+```
+
+Update your @env.local@ file with the following:
+
+```
+POSTGRES_URL="postgres://postgres:postgres@localhost:5432/postgres"
+POSTGRES_PRISMA_URL="postgres://postgres:postgres@localhost:5432/postgres"
+POSTGRES_URL_NON_POOLING="postgres://postgres:postgres@localhost:5432/postgres"
+POSTGRES_USER=postgres
+POSTGRES_HOST=localhost
+POSTGRES_PASSWORD=postgres
+POSTGRES_DATABASE=postgres
+SESSION_SECRET=<SOME_SECRET>
+NEXT_PUBLIC_ABLY_API_KEY=<YOUR_ABLY_API_KEY>
+NEXT_PUBLIC_ABLY_CHANNEL_NAMESPACE=<SOME_NAMESPACE>
+```
+
+- Replace @<SOME_SECRET>@ with some random string.
+- Replace @<YOUR_ABLY_API_KEY>@ with your Ably API Key
+- Optionally replace @<SOME_NAMESPACE>@ with a string value used as a [namespace](https://faqs.ably.com/what-is-a-channel-namespace-and-how-can-i-use-them) for the Pub/Sub channel being used by the model.
+
+Export the environment variables in your shell session:
+
+```[sh]
+export $(grep -s -v "^#" env.local | xargs) # export environment variables
+```
+
+h3(#step-3). Running Docker container
```

Comment:
why docker container and not integration rule?

Reply:
I've brought this up as a discussion in Slack